app/models/application.rb

```diff
@@ -0,0 +1,7 @@
+class Application < ApplicationRecord
+  belongs_to :user
+  belongs_to :scholarship
+  # before_save only verified members can submit
```

Comment:
This would end up being a validation on the user's validated attribute.

You can read about rails validations here: http://edgeguides.rubyonrails.org/active_record_validations.html#validation-helpers